pages/api/rating.ts

```diff
@@ -0,0 +1,22 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { IUserMailResponse } from "../../types/Mail";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<IUserMailResponse>
+) {
+  try {
+    //TODO: add query validation later
+    const user = req.query.mail;
+    const rating = req.query.rating;
```

Comment:
You could packed this data in single object for example:
`data = {
user: req.query.mail,
rating: req.query.rating
}`